types/verifier.go

```diff
@@ -46,3 +49,25 @@ func VerifySignature(sigAccAddress sdk.AccAddress, sigHash []byte, sig []byte) e
 
 	return nil
 }
+
+func VerifyBlsSignature(blsPubKey bls.PublicKey, sigHash [32]byte, blsSig []byte) error {
```

Comment:
I think the code here is duplicated, as the cosmos-sdk already use bls, so please reuse the code there.

Reply:
@alexgao001  please take a look.